examples/test/test_i_multiple_kms_cmk_regions.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Unit test suite for the encryption and decryption using multiple KMS CMKs in multiple regions example."""
+
+import botocore.session
+import pytest
+
+from ..src.multiple_kms_cmk_regions import multiple_kms_cmk_regions
+
+# from .examples_test_utils import get_cmk_arn
+from .examples_test_utils import static_plaintext
+
+
+pytestmark = [pytest.mark.examples]
```

Comment:
Should this be used as a decorator?

Reply:
We could, but I'd prefer to keep it like this. `pytestmark` takes an iterable of markers and applies all of them to every test in the module.
In this particular case, there is only the one test, but I'd prefer to stick with this mechanism for consistency.